libs/designsystem/src/lib/components/list/list.component.html

```diff
@@ -1,5 +1,106 @@
+<!-- VIRTUAL SCROLL LIST -->
+<ng-container *ngIf="isVirtualScrollEnabled">
+  <ion-virtual-scroll
+    *ngIf="!isSectionsEnabled"
+    [items]="items"
+    class="list"
+    [trackBy]="trackBy || defaultTrackBy"
+    kirbyInfiniteScroll
+    [itemHeight]="itemHeightFn"
+    (scrollEnd)="onLoadOnDemand()"
+    [disabled]="!isLoadOnDemandEnabled"
+    [class.has-header]="headerTemplate"
+    [headerFn]="headerFn"
+    [footerFn]="footerFn"
+    [class.has-footer]="footerTemplate"
+    [class.has-divider]="showDivider"
+    (window:resize)="onResize()"
+    #list
+  >
+    <ion-list-header *virtualHeader="let header">
+      <ng-container *ngTemplateOutlet="headerTemplate"></ng-container>
+    </ion-list-header>
+
+    <ion-item-sliding
+      *virtualItem="let item"
+      [class.selected]="isSelectable && item === selectedItem"
+      class="{{ getItemColor(item) }}"
+      [disabled]="isSwipingEnabled ? null : true"
+      [class.first]="isFirst(item)"
+      [class.last]="isLast(item)"
+      keyHandler
+      (click)="isSelectable ? onItemSelect(item) : null"
+    >
+      <ng-container
+        *ngTemplateOutlet="swipeActionsTemplate; context: { $implicit: item }"
+      ></ng-container>
+      <ng-container
+        *ngTemplateOutlet="defaultItemTemplate; context: { $implicit: item }"
+      ></ng-container>
+    </ion-item-sliding>
+
+    <div *virtualFooter="let footer" class="footer">
+      <ng-container *ngTemplateOutlet="footerTemplate"></ng-container>
+    </div>
+  </ion-virtual-scroll>
+
+  <!-- Virtual scroll list with sections -->
+  <ion-virtual-scroll
+    *ngIf="isSectionsEnabled && sortedItems"
+    [items]="sortedItems"
+    class="list"
+    [trackBy]="trackBy || defaultTrackBy"
+    kirbyInfiniteScroll
+    [itemHeight]="itemHeightFn"
+    (scrollEnd)="onLoadOnDemand()"
+    [disabled]="!isLoadOnDemandEnabled"
+    [class.has-header]="headerTemplate"
+    [headerFn]="sectionHeaderFn"
+    [footerFn]="footerFn"
+    [class.has-footer]="footerTemplate"
+    [class.has-divider]="showDivider"
+    (window:resize)="onResize()"
+    #list
+  >
+    <ng-container *virtualHeader="let header">
+      <ion-item-divider>
+        <ng-container
+          *ngTemplateOutlet="sectionHeaderTemplate; context: { $implicit: header }"
+        ></ng-container>
+      </ion-item-divider>
+    </ng-container>
+
+    <ion-item-sliding
+      *virtualItem="let item; let index = index"
+      [class.selected]="isSelectable && item === selectedItem"
+      class="{{ getItemColor(item) }}"
```

Comment:
Fixing expand/collapse on transaction list upcoming transactions

Reply:
Items are being reused now, so adding a class with the renderer will persist even after the items change.